support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -261,75 +241,34 @@ export function ThreeTierCard({
 					href={link}
 					cssOverrides={btnStyleOverrides}
 					data-qm-trackable={quantumMetricButtonRef}
-					aria-label={label}
+					aria-label={title}
 				>
 					{ctaCopy}
 				</LinkButton>
 			</ThemeProvider>
 
-			{benefitsSummary && (
-				<div css={benefitsPrefixCss}>
-					<span>
-						{benefitsSummary.map((stringPart) => {
-							if (typeof stringPart !== 'string') {
-								return <strong>{stringPart.copy}</strong>;
-							} else {
-								return stringPart;
-							}
-						})}
-					</span>
-				</div>
-			)}
-			{offersSummary && (
+			{product === 'TierThree' && (
 				<div css={benefitsPrefixCss}>
 					<span>
-						{offersSummary.map((stringPart) => {
-							if (typeof stringPart !== 'string') {
-								return <strong>{stringPart.copy}</strong>;
-							} else {
-								return stringPart;
-							}
-						})}
+						The rewards from <strong>All-access digital</strong>
 					</span>
+					{benefits.length > 0 && <span css={benefitsPrefixPlus}>plus</span>}
 				</div>
 			)}
-			{(benefitsSummary ?? offersSummary) && (
-				<span css={benefitsPrefixPlus}>plus</span>
-			)}
 			<BenefitsCheckList
-				benefitsCheckListData={benefits
-					.filter((benefit) => filterBenefitByRegion(benefit, countryGroupId))
-					.filter((benefit) => filterBenefitByABTest(benefit, abParticipations))
-					.map((benefit) => {
-						return {
-							text: benefit.copy,
-							isChecked: true,
-							toolTip: benefit.tooltip,
-							isNew: benefit.isNew,
-							hideBullet: benefit.hideBullet,
-						};
-					})}
+				benefitsCheckListData={benefits.map((benefit) => {
+					return {
+						text: benefit.copy,
+						isChecked: true,
+						toolTip: benefit.tooltip,
+						label: benefit.label?.copy,
+						hideBullet: benefits.length <= 1 && product !== 'TierThree',
```

Comment:
this is for the recurring contributions "benefit", it would be nice if we didn't have to do this!